### PR TITLE
Storefront style guide and styles fixes

### PR DESCRIPTION
**Pull request by weronka**
- add storefront style guide page
- reorganize styles files
- fix class names and make styles more consistent

Ref #957

### CODE REVIEW

**Comment by benekex2** [comment]
- [ ] We need to show disabled states for buttons and form elements
- [ ] Select problems on Chrome

**Comment by patrys** [comment]
The above select should be replaced and merged with a global country selector. We already have a concept of user's country (defaulting to GeoIP) and it's used to convert prices to a local currency. There's no reason to have a separate selector just for shipping estimates.

I think it makes more sense to say:

>Shipping to Poland: $10
>Shipping elsewhere? ⟪Change country⟫

**Comment by benekex2** [comment]
We should add:
- [ ] CSS classes and helpers - useful from bootstrap and those that we added
- [ ] content to tabs to show their functionality
- [ ] product list example with sale flag
- [ ] product image slider (?)
- [ ] button on image (homepage blocks)
- [ ] all used icons and hover effects

File: saleor/static/scss/storefront/components/_cards.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    ul.icons {
+      padding-left: 0;
+      padding-top: $global-padding;
+      list-style: none;
+      margin-bottom: 0;
+      li {
+        width: 49%;
+        display: inline-block;
+        a {
+          line-height: 18px;
```
Comment (benekex2) [inline saleor/static/scss/storefront/components/_cards.scss]: use rem units

File: saleor/static/scss/storefront/components/_filters.scss [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    font-weight: bold;
+    cursor: pointer;
+    .collapse-filters-icon {
+      float: right !important;
+      width: 20px;
+      height: 20px;
+    }
+  }
+  ul {
+    padding: $global-padding/2 0;
```
Comment (benekex2) [inline saleor/static/scss/storefront/components/_filters.scss]: missing space $global-padding / 2

File: templates/userprofile/address-edit.html [hunk 3]
```diff
@@ -34,18 +34,18 @@
     <div class="col-sm-8 col-lg-6 offset-sm-2 offset-lg-3">
         <form method="post" novalidate>
             {% csrf_token %}
-            <h1 class="text-xs-center">
+            <h2 class="text-xs-center">
```
Comment (benekex2) [inline templates/userprofile/address-edit.html]: remove xs flag

**Pull request merged**